### ISSUE TITLE
Canvas height doesn't expand to fill viewport in annotation track with autoHeight

### ISSUE THREAD

**Issue by zareifar** [igvteam/igv.js#1264]
Dear IGV team,

I am using IGV version 2.7.4 and in my IGV browser, before I set the annotation track autoHeight to true, everything worked fine. However, now that the height of the annotation track is set to auto, for some locations, the canvas height doesn't expand to fill the viewport and results in genes not being visible. As an example, you can use the code below the recreate the problem:

```<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <link rel="shortcut icon" href="https://igv.org/web/img/favicon.ico">
    <script src="https://cdn.jsdelivr.net/npm/igv@2.7.4/dist/igv.min.js"></script>
    <title>igv.js</title>
</head>

<body>

<h1>Annotation track autoHeight issue</h1>

<div id="igvDiv" style="padding-top: 10px;padding-bottom: 10px; border:1px solid lightgray"></div>

<script type="text/javascript">

    document.addEventListener("DOMContentLoaded", function () {

      var options =
            {
                genome: "hg38",
                locus: "chr1:7,663,877-7,663,917",
                trackDefaults: {
                    annotation: {
                        autoHeight: true
                    }
                }
            };

        var igvDiv = document.getElementById("igvDiv");

        igv.createBrowser(igvDiv, options)
            .then(function (browser) {
                console.log("Created IGV browser");
            })
    })


</script>
</body>
</html>

**Comment by jrobinso** [comment]
This should be fixed now, to be released next week.

**Comment by jrobinso** [comment]
This is released now as 2.7.5
